hack/git-chglog/config.yaml

```diff
@@ -6,7 +6,7 @@ style: github
 template: CHANGELOG.tpl.md
```

Comment:
shouldn't this folder be removed, `gi-chglog`

Reply:
yep, nice catch